forms-flow-web/src/components/Form/constants/formSearchValicdation.js

```diff
@@ -0,0 +1,18 @@
+
+const searchValidator = (searchText) => {
```

Comment:
same here. You can use the same logic but try to maintain better coding standards. no need to use var anymore.  Remove unused code, follow formatting rules, etc.  Write test cases for utility functions.